src-db/database/sourcedata/AD_ELEMENT.xml

```diff
@@ -56,6 +56,20 @@
 <!--19CDF74F92D04F8DA1098F84509C3EC4-->  <ISGLOSSARY><![CDATA[N]]></ISGLOSSARY>
 <!--19CDF74F92D04F8DA1098F84509C3EC4--></AD_ELEMENT>
 
+<!--1A2C25FCE4454945B5A5D084C815FA85--><AD_ELEMENT>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <AD_ELEMENT_ID><![CDATA[1A2C25FCE4454945B5A5D084C815FA85]]></AD_ELEMENT_ID>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <COLUMNNAME><![CDATA[Lineno]]></COLUMNNAME>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <NAME><![CDATA[Line No.]]></NAME>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <PRINTNAME><![CDATA[Line No.]]></PRINTNAME>
+<!--1A2C25FCE4454945B5A5D084C815FA85-->  <DESCRIPTION><![CDATA[This column is the line number of messages in a Copilot chat. ]]></DESCRIPTION>
```

Comment:
I think that Help and Description fields should be swapped.

 Description should be at a more technical level, the user does not sees it.
 Help should be more user-friendly, the user sees it when going into the 'Help' section.

Does it makes sense?